### PR TITLE
Make it clear to what {base} is referring to in SaaS and Manged envs

### PR DESCRIPTION
**Pull request by moschinski**

### CODE REVIEW

File: metric-example/query-by-example.md
Comment (myieye) [inline metric-example/query-by-example.md:11]:
```suggestion
*   the host and path of the Dynatrace instance you want to run metric queries against, below referred to as `{base}` 

```

**Review by myieye — APPROVED** [review]
Unless you've already reached out to someone else to merge this, it would be nice to know if you're expecting this to just get merged or are waiting for an approval from someone more familiar with the content.

**Comment by moschinski** [comment]
@myieye the changes can be merged, no further reviews needed.

**Pull request merged**
